src/components/FormFieldComponent.vue

```diff
@@ -318,8 +264,14 @@ export default {
     }
   },
   computed: {
+    id () {
+      return this.field.id
+    },
```

Comment:
Probably my lack of understanding of Vue or the application, isn't computed for variables that can change or are calculated (and you don't want to calculate every rerender)? I feel like id makes more sense on the data property of the component.

Reply:
turns out it changes from undefined to defined at some point. so computed is necessary